src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.CustomeException;
```

Comment:
The import statement for `CustomeException` contains a typo. It should be `CustomException`.